packages/lesswrong/components/walledGarden/GardenCodesList.tsx

```diff
@@ -4,21 +4,37 @@ import { useMulti } from '../../lib/crud/withMulti';
 import { Components, registerComponent } from '../../lib/vulcan-lib';
 import { useCurrentUser } from '../common/withUser';
 
-export const GardenCodesList = ({classes, terms}:{classes:ClassesType, terms: any}) => {
+export const GardenCodesList = ({classes, limit, personal=false}:{
+  classes:ClassesType,
+  limit?: number,
+  personal?: boolean}) => {
```

Comment:
Style nitpick: when you have any sort of block which spans multiple lines, the start and end delimiter should usually be on lines that start in the same column. So this would be
```
export const GardenCodesList = ({classes, limit, personal=false}:{
  classes:ClassesType,
  limit?: number,
  personal?: boolean
}) => {
```
This avoids making it look like the `personal?: boolean` line shares a block/indent-level with the `const { GardenCodesItem } = Components` line below it.

Reply:
Yep, makes sense! Will fix in next PR.